docs/ums/README.md

```diff
@@ -153,12 +224,26 @@ You can do user management via CRUD operations. This method is mainly for the cu
 ::: warning
 This will be deprecated in future versions.
 :::
-<CodeSwitcher :languages="{js:'JavaScript',bash:'cURL'}">
+<CodeSwitcher :languages="{js:'JavaScript',py:'Python',bash:'cURL'}">
 <template v-slot:js>
 
 ``` js
-ums.deleteUser('Elon@tesla.com', password); 
+ums.deleteUser('Elon@tesla.com', password)
+.subscribe(user => {}, error=>{});    
+```
+</template>
+<template v-slot:py>
+
+``` py
+...
+  res = client.request(
+          method='DELETE',
+          url='/ums/user/'
+        ) 
+  print(res)
```

Comment:
will return nothing here, you can remove this line

Reply:
fixed